src/content/tutorials/index.md

```diff
@@ -34,10 +32,24 @@ which are fundamental to most Dart programs.
 </div>
 
 
-## Server-side Dart tutorials
+<a id="server-side-dart-tutorials" aria-hidden="true"></a>
+## Build server-side apps {:#server}
 
 {% include 'server-tutorials.md' %}
 
-## More tutorials
+<a id="more-tutorials" aria-hidden="true"></a>
+## Keep learning
 
-Check out the [Flutter codelabs and tutorials]({{site.flutter-docs}}/codelabs).
+Follow more tutorials and codelabs to get more hands-on experience
```

Comment:
**issue:** Same as [line 2778](https://github.com/dart-lang/site-www/pull/5858/files#diff-75ed5ce4cf7ea66ddd2f5aa89546afe28498b1afb33bcf6a6f56fb7407445cd6R2778) in [js-to-dart.md](src/content/resources/coming-from/js-to-dart.md)

```suggestion
To get more hands-on experience, try more of the available tutorials
```